flexy-osisoft-connector/web/js/FlexyConnector.js

```diff
@@ -0,0 +1,260 @@
+function saveJSON()
+{
+   waitingDialog.show("Sending configuration to the flexy");
+
+   //make an empty object
+   var jsonObject = {};
+
+   //Fetch the ServerConfig
+   var IPVal = document.getElementById("IP").value;
+   var WebID = document.getElementById("WebID").value;
+   var Credentials = document.getElementById("Credentials").value;
+   jsonObject.ServerConfig = { IP: IPVal, WebID: WebID, Credentials: Credentials };
+
+   //Fetch the eWONConfig
+   var CertificatePath = document.getElementById("CertificatePath").value;
+   jsonObject.eWONConfig = { CertificatePath: CertificatePath };
+
+   //Fetch the AppConfig
+   var CycleTimeMs = document.getElementById("CycleTimeMs").value;
+   var PostDuplicateTagValues = document.getElementById("PostDuplicateTagValues").value;
+   jsonObject.AppConfig = { CycleTimeMs: CycleTimeMs, PostDuplicateTagValues: PostDuplicateTagValues };
+
+   //Fetch the TagList
+   jsonObject.TagList = [];
+   var tagTableRef = document.getElementById('tagTable').getElementsByTagName('tbody')[0]

```

Comment:
While not needed you are missing semi-colon here. May want to add it for consistency.
BTW, I think the ESLint plugin in VSCode can warn you about this if configured to care about it.

I see some other occurrences below as well

Reply:
fixed